frontend/src/components/Common/UserInput.css

```diff
@@ -1,9 +1,11 @@
 .JoinGroup {
     outline: none;
-    width: 25vw;
-    border-radius: 30px;
-    border: 4px solid #000000;
-    font-size: 2.5vw;
+    background-color: #242424;
+    color: rgb(216, 208, 208);
```

Comment:
I noticed that colouring system is pretty inconsistent,
Above, it uses "color:white"
I think it would be better if the colouring system all used the same format for consistency. Possibly the rgb (x, y, z) format.